charts/opentelemetry-kube-stack/templates/hooks.yaml

```diff
@@ -0,0 +1,56 @@
+{{- if .Values.cleanupJob.enabled }}
+apiVersion: v1
+kind: ServiceAccount
```

Comment:
Do the `ServiceAccount`, `Role`, and `RoleBinding` need something to make sure they get cleaned up on `helm uninstalled` as well?

Reply:
argh yeah they probably need more hooks 🙃

Reply:
I hate helm hooks.